client/dive-common/components/AttributeInput.vue

```diff
@@ -143,6 +154,23 @@ export default defineComponent({
       @change="change"
       @keydown="onInputKeyEvent"
     >
+    <div
+      v-else-if="datatype === 'number' && (typeSettings && typeSettings.type ==='slider')"
+    >
+      <div class="slider-label">
+        {{ value }}
+      </div>
+      <v-slider
+        :value="value"
+        :step="typeSettings.steps ? typeSettings.steps
+          : (typeSettings.range[1] - typeSettings.range[0])/2.0"
```

Comment:
It's possible for both `typeSettings.steps` and `typeSettings.range` to be undefined and the slider breaks. Probably best to make one or the other mandatory when creating the slider.
![Screenshot from 2022-08-16 13-30-58](https://user-images.githubusercontent.com/40494088/184943062-e247aae2-6c15-4236-8088-a40ac48c9264.png)

 
![Screenshot from 2022-08-16 13-30-37](https://user-images.githubusercontent.com/40494088/184943086-7b2e482d-3323-49ad-880f-5ee4ae4999d4.png)